1.2.Javascript and Web.md~

```diff
@@ -0,0 +1,6 @@
+###জাভাস্ক্রিপ্ট এবং ওয়েব ### 
```

Comment:
I guess this is the backup file generated by your text editor. Were you using Gedit? Using a text editor that has live preview for Markdown would help.